src/App.js

```diff
@@ -62,11 +67,19 @@ class App extends Component {
     };
   };  // End constructor()
 
+  /** Set the language of the app
+   * @param {object} evnt - An event object, which is not actually used in the function but is passed in by Semantic UI React input components
```

Comment:
Nice! I like this description!

Reply:
Which is weird, because this is _also_ a description of React behavior...

Reply:
Lol to be fair, it is a description of Semantic UI React behavior, so that's a little more specific than just general React basics!